lib/components/viewers/nearby/nearby-view.tsx

```diff
@@ -286,6 +306,32 @@ function NearbyView({
       >
         {/* This is used to scroll to top */}
         <div aria-hidden ref={firstItemRef} />
+        <LocationField
+          className="nearby-view-location-field"
+          geocoderConfig={geocoderConfig}
+          getCurrentPosition={getCurrentPosition}
+          inputPlaceholder={intl.formatMessage({
+            id: 'components.NearbyView.searchNearby'
+          })}
+          location={{
+            // Provide a 0 default in case the nearby view coords are null
+            lat: 0,
+            lon: 0,
+            ...nearbyViewCoords,
+            name: reversedPoint
+          }}
+          LocationIconComponent={() => (
+            <Search style={{ marginRight: 5, padding: 5 }} />
+          )}
+          locationType="to"
+          onLocationSelected={(selection) => {
+            const { location } = selection
+            setViewedNearbyCoords(location)
+            map && zoomToPlace(map, location)
+            setReversedPoint(location.name || '')
+          }}
+          sortByDistance
+        />
         {loading && (
```

Comment:
Loading spinner is showing up on the location field which is a little misleading? Could we position it down to the results container

![image](https://github.com/user-attachments/assets/bb8d044b-f10f-41bd-99b2-c5ff45a137a8)